### ISSUE TITLE
Ability to set download directory and disable download or open popup using Firefox

### ISSUE THREAD

**Issue by pxfurkanenes** [mherrmann/selenium-python-helium#19]
Helium really does a great job simplifying selenium interface and I like using it. However, I couln't manage to set a profile for Firefox in order to change download directory and disable download popup. I think these were essential features for me while using selenium. Are you planning to add this kind of settings to `start_firefox` and `start_chrome` just like `headless=True` option?

**Comment by mherrmann** [comment]
Thank you for the kind words. I'm happy to hear you like Helium.

Regarding your question: Helium lets you supply `ChromeOptions` to `start_chrome` since the [last release](https://github.com/mherrmann/selenium-python-helium/releases/tag/v3.0.2). Maybe something similar could be added for Firefox?

Either way, I won't have time to implement this. But I will be happy to merge a PR that does it.

**Comment by ftnext** [comment]
I implemented this feature. See PR #22 .

### Usage

```python
from selenium.webdriver import FirefoxOptions

options = FirefoxOptions()
options.set_preference("browser.download.useDownloadDir", True)
options.set_preference("browser.download.folderList", 2)
options.set_preference("browser.download.dir", "path/to/download/directory")
options.set_preference("browser.helperApps.neverAsk.saveToDisk", "text/csv")  # specify content-type
start_firefox("connpass.com/login", options=options)
```

ref: https://stackoverflow.com/a/36309735

**Comment by mherrmann** [comment]
Thank you again @ftnext. I've released this on PyPI as version 3.0.3.
